packages/react-component-library/src/components/DatePicker/Input.tsx

```diff
@@ -0,0 +1,197 @@
+import { IconEvent } from '@royalnavy/icon-library'
+import React, { memo, useCallback } from 'react'
+
+import { CALENDAR_TABLE_VARIANT } from './constants'
+import { DATE_FORMAT } from '../../constants'
+import { DATEPICKER_ACTION } from './types'
+import { DatePickerProps } from './DatePicker'
+import { hasClass } from '../../helpers'
+import { InlineButton } from '../InlineButtons/InlineButton'
+import { isDateValid } from './utils/isDateValid'
```

Comment:
Import can be shortened here:
```suggestion
import { isDateValid } from './utils'
```